dev/drivers/scripts/stats/analyses/jevs_analyses_ccpa_precip_stats.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/bash
+#PBS -N jevs_analyses_ccpa_precip_stats
+#PBS -j oe
+#PBS -S /bin/bash
+#PBS -q "dev"
+#PBS -A VERF-DEV
+#PBS -l walltime=00:30:00
+#PBS -l place=shared,select=1:ncpus=1:mem=10GB
+#PBS -l debug=true
+#PBS -V
+ 
+set -x
+
+cd $PBS_O_WORKDIR
+
+export model=evs
+
+export HOMEevs=/lfs/h2/emc/vpppg/noscrub/$USER/EVS_analyses_precip/EVS
```

Comment:
@PerryShafran-NOAA , can you update your drivers to revert HOMEevs back to the default value? I will make the change in my local checkout, but we need it changed before merging.

Reply:
Yes, done.  I can't believe that I forgot to remove the testing directory.